app/Jobs/ApplicationDeploymentJob.php

```
@@ -2422,7 +2470,398 @@ private function generate_nixpacks_env_variables()
+        return 'docker buildx create --name coolify-railpack --driver docker-container 2>/dev/null || true'
+            ." && {$environmentPrefix}docker buildx build --builder coolify-railpack"
+            ." {$this->addHosts} --network host"
+            ." --build-arg BUILDKIT_SYNTAX=\"{$frontendImage}\""
+            ." {$cacheArgs}"
+            ."{$secretFlags}"
+            .' -f /artifacts/railpack-plan.json'
+            .' --progress plain'
+            .' --load'
+            ." -t {$imageName}"
+            ." {$this->workdir}";
+    }
```

PHPMD (2.15.0)

[error] The method railpack_build_command is not named in camelCase. (undefined)

(CamelCaseMethodName)